### PR TITLE
Fix orphan Job().result_proxy reference

### PR DESCRIPTION
**Pull request by brianjmurrell**
Job().result_proxy was removed in ed4dcd1 however references to it
remained.

This removes one of the remaining references.

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

**Review by beraldoleal — APPROVED** [review]
Unless I'm missing something, LGTM.

**Comment by willianrampazzo** [comment]
This is proposing a change on an unsupported and old version of Avocado LTS.

**Comment by beraldoleal** [comment]
> Unless I'm missing something, LGTM.

Just realized the 52lts. We have 2 LTS after 52 already. So AFAICT we don't support this anymore.

**Comment by brianjmurrell** [comment]
Where would I find the document that says what is currently supported and what is not, so that I file a bug for a new version in EPEL?

**Comment by willianrampazzo** [comment]
> Where would I find the document that says what is currently supported and what is not, so that I file a bug for a new version in EPEL?

You can check how we support releases here: https://avocado-framework.readthedocs.io/en/83.0/releases/index.html?highlight=supported#how-we-release-avocado

**Comment by brianjmurrell** [comment]
@willianrampazzo Thanks.  It's a little wishy-washy though:

> bugfixes are backported on demand for a period of _about_ 18 months after the release

Granted, even with the _fuzzy_ **about** in that description, it's probably safe to say that nobody would argue about 52.x being older than that.  But for a release that is 19 or even maybe 20 months since created, one could get into quite a debate about whether 19 or 20 were _about_ 18 or not.

Ultimately, it would be more useful if there was a definitive supported/unsupported indicator next to each LTS release on that page.

I will see what the EPEL maintainer has to say...

**Comment by willianrampazzo** [comment]
> @willianrampazzo Thanks. It's a little wishy-washy though:
> 
> > bugfixes are backported on demand for a period of _about_ 18 months after the release
> 
> Granted, even with the _fuzzy_ **about** in that description, it's probably safe to say that nobody would argue about 52.x being older than that. But for a release that is 19 or even maybe 20 months since created, one could get into quite a debate about whether 19 or 20 were _about_ 18 or not.

That is exactly the intent. As an example, if the project notices that users will not migrate from an LTS version due to some software dependency, we may add some months of support to it. Making the dates strict does not help users.

> 
> Ultimately, it would be more useful if there was a definitive supported/unsupported indicator next to each LTS release on that page.

I agree, listing what version is currently supported makes things clear.

**Comment by brianjmurrell** [comment]
> That is exactly the intent.

I agree with and appreciate this policy as long as the moment you all decide that an LTS is beyond even the _about_ stage, you transition some **supported** flag to **unsupported**.

That will make having external (to your team) discussions easier without having to drag you folks into a debate about whether something is still within the _about_ window.

**Comment by willianrampazzo** [comment]
> > That is exactly the intent.
> 
> I agree with and appreciate this policy as long as the moment you all decide that an LTS is beyond even the _about_ stage, you transition some **supported** flag to **unsupported**.
> 
> That will make having external (to your team) discussions easier without having to drag you folks into a debate about whether something is still within the _about_ window.

I totally agree with you. I have opened issue #4348 to track this documentation.

**Comment by beraldoleal** [comment]
@brianjmurrell @willianrampazzo since this is about 52lts and there is already an open issue to address the problem, I'm closing this. Please feel free to reopen if you would like to push this and/or have more arguments to add.

**Pull request closed without merging**
